test/bin/manage_vm_storage_pool.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+#
+# This script should be run on the hypervisor to set up storage pool for VMs.
+
+set -euo pipefail
+
+SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+source "${SCRIPTDIR}/common.sh"
+
+usage() {
+    cat - <<EOF
+${BASH_SOURCE[0]} (create|cleanup)
+
+  -h           Show this help.
+
+create: Set up a storage pool with the given name and dir. Uses
+        variables VM_STORAGE_POOL and VM_DISK_DIR.
+
+cleanup: Remove storage pool rules created by 'create' command.
+
+EOF
+}
+
+action_create() {
+    sudo sh -c '
+virsh pool-define-as '"${VM_STORAGE_POOL}"' dir --target '"${VM_DISK_DIR}"'
+virsh pool-build '"${VM_STORAGE_POOL}"'
+virsh pool-start '"${VM_STORAGE_POOL}"'
+virsh pool-autostart '"${VM_STORAGE_POOL}"'
```

Comment:
This shouldn't affect the result very much, but because of no `set -e` all these commands will run even if first fails. 
Although without `set -x` it might be hard to match stderr to command. 
Maybe they should be at least chained with `&&`?